server/CMakeLists.txt

```diff
@@ -91,6 +91,28 @@ install(
   NAMELINK_COMPONENT ${IRODS_PACKAGE_COMPONENT_DEVELOPMENT_NAME}
 )
 
+# Initialize IRODS_CONFIGURATION_SCHEMA_PREFIX used by configuration schemas.
+# This variable helps to produce the correct URI for the "$id" JSON property.
+if (CMAKE_INSTALL_PREFIX STREQUAL "/")
+  # This branch is primarily for packaged installations, but can work for non-package
+  # installs as well.
+  set(IRODS_CONFIGURATION_SCHEMA_PREFIX "")
+elseif (NOT CMAKE_INSTALL_PREFIX STREQUAL "")
+  # This branch is for non-package installations.
```

Comment:
This is not a good way to detect non-package installs, but I think the behavior here is probably correct.